components/console/helpers/table.rst

```diff
@@ -179,17 +192,24 @@ The output of this command will be:
 
     Support for vertical rendering was introduced in Symfony 6.1.
 
+Customizing the Table Style
+---------------------------
+
 The table style can be changed to any built-in styles via
 :method:`Symfony\\Component\\Console\\Helper\\Table::setStyle`::
 
-    // same as calling nothing
+    // this 'default' style is the one used when no style is specified
     $table->setStyle('default');
 
-    // changes the default style to compact
+Built-in Table Styles
+~~~~~~~~~~~~~~~~~~~~~
+
+**Compact**::
+
     $table->setStyle('compact');
     $table->render();
 
-This code results in:
+This outputs:
```

Comment:
I saw we don't have same wording for result in this file, should we also update them ?

`This outputs:`
`The output of this command will be:`
`This generates:`
`This will display the following table in the terminal:`
`This will display:`
Nothing for `Adding Table Titles` section

Reply:
Nice catch, streamlining those things would be nice.

WDYT @symfony/docs ?

Reply:
Good catch! I made all this consistent and used `This outputs` because it's very concise and easy to understand. Thanks!